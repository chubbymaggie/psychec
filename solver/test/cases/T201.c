void f()
{
    char const *group;
    xstrdup (pwd ? pwd->pw_name
                 : imaxtostr (uid, buf) ? imaxtostr (uid, buf)
                                        : umaxtostr (uid, buf));
    if (user) {}
    else if (group)
    {
        xstrdup (group);
    }
}
