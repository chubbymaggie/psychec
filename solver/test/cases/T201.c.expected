/* Forward declarations */
typedef  struct TYPE_2__   TYPE_1__ ;

/* Type definitions */
struct TYPE_2__ {char const* pw_name; } ;

/* Variables and functions */
 int /*<<< orphan*/  buf ;
 char const* imaxtostr (int /*<<< orphan*/ ,int /*<<< orphan*/ ) ;
 TYPE_1__* pwd ;
 int /*<<< orphan*/  uid ;
 char const* umaxtostr (int /*<<< orphan*/ ,int /*<<< orphan*/ ) ;
 scalar_t__ user ;
 int /*<<< orphan*/  xstrdup (char const*) ;

